libs/langchain/langchain/cache.py

```diff
@@ -216,10 +216,25 @@ def __init__(self, database_path: str = ".langchain.db"):
 class RedisCache(BaseCache):
     """Cache that uses Redis as a backend."""
 
-    # TODO - implement a TTL policy in Redis
-
-    def __init__(self, redis_: Any):
-        """Initialize by passing in Redis instance."""
+    def __init__(self, redis_: Any, *, ttl: Optional[int] = None):
```

Comment:
nit:  https://google.github.io/styleguide/pyguide.html#383-functions-and-methods -- we follow google style for doc-strings

Parameters -> Args: